google-cloud-bigquery/AUTHENTICATION.md

```diff
@@ -67,12 +67,14 @@ The environment variables that BigQuery checks for credentials are configured on
 5. `GOOGLE_APPLICATION_CREDENTIALS` - Path to JSON file
 
 ```ruby
+require "googleauth"
```

Comment:
Here and other like this with env vars -- it's ok to leave as-is, no need to change them